README.md

```diff
@@ -74,10 +94,20 @@ This project is built using the [Flask](http://flask.pocoo.org/) web framework,
   https://<your-ngrok-subdomain>.ngrok.io/answer
   ```
 
+That's it!
```

Comment:
add one more step to say "Call your Twilio number to test it out" or something to that effect